src/Servers/IIS/AspNetCoreModuleV2/CommonLib/CommonLib.vcxproj

```diff
@@ -75,20 +75,15 @@
   <PropertyGroup Label="UserMacros" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'">
     <LinkIncremental>true</LinkIncremental>
-    <OutDir>$(MSBuildProjectDirectory)\bin\$(Configuration)\$(Platform)\</OutDir>
```

Comment:
Going forward, will VS restore these settings when we edit project settings? I've seen it undo changes similar to this&hellip;

Reply:
I opened VS and built the project and didn't see .vcxproj files change. What would trigger VS to undo this change?

Reply:
Change something in the project's property page while in VS. If that update to the project doesn't regurgitate these settings, life is grand.

Reply:
Seems to be okay. Didn't run into problems with VS when changing properties via UI.